packages/wrangler/turbo.json

```diff
@@ -21,55 +21,61 @@
 				"config-schema.json"
 			],
 			"env": [
```

Comment:
this is the list of environment variables a task depends on. Any change to these environment variables will cause all tasks to miss cache

Reply:
The ones listed here in `env` are actually used by the `scripts/bundle.ts` file to inject code into the Wrangler build output. So it is important that these are part of the cache key.